releasenotes/notes/experiment-artifacts-c481f4e07226ce9e.yaml

```
@@ -0,0 +1,20 @@
+---
+features:
+  - |
+    An artifact class has been introduced to store long-form data generated by experiments.
+    The :class:`.CurveFitResult` and :class:`.ScatterTable` generated by experiments 
+    are now stored in artifacts in the :class:`.ExperimentData` class. :meth:`.add_artifacts`
+    and :meth:`.delete_artifact` have been added to manipulate the artifacts. These will be uploaded
+    to the cloud service in JSON form along with the rest of the :class:`.ExperimentData` object
+    when saved. For more information, see the :doc:`artifacts how-to </howtos/artifacts>`.
+upgrade:
+  - |
+    Direct access to the curve fit summary in :class:`.ExperimentData` has moved from 
+    :meth:`.analysis_results` to :meth:`.artifacts`, where values are stored in
+    :meth:`.artifacts.data`. For example, to access the chi-squared of the fit,
```

Does this work? It looks funny that it renders as `data()` in this line and then `data.chisq` on the next line.

```suggestion
    :attr:`.artifacts.data`. For example, to access the chi-squared of the fit,
```

---

Thanks for catching this, updated.